src/main/java/pl/javastart/task/phone/Phone.java

```diff
@@ -0,0 +1,27 @@
+package pl.javastart.task.phone;
+
+import pl.javastart.task.contracts.Contract;
+
+public class Phone {
+    Contract contract;
```

Comment:
Dodałbym private. Ogólnie traktowałbym private jako specyfikator wyjściowy, chyba że masz inne klasy w tym pakiecie, który powinny mieć dostęp do tego pola, to wtedy jest uzasadnione, żeby nieco poluźnić te restrykcje